### PR TITLE
Include additional blocks for space when counting total block quota

### PR DESCRIPTION
**Pull request by Devorein**
### WHY

[Card](https://app.charmverse.io/charmverse/create-spaces-for-op-projects-with-additional-free-blocks-37654345295167957)

### CODE REVIEW

File: components/settings/subscription/hooks/useSpaceSubscription.ts [hunk 1]
```diff
@@ -31,7 +34,8 @@ export function useSpaceSubscription() {
     [spaceSubscription?.status]
   );
 
-  const spaceBlockQuota = defaultFreeBlockQuota * 1000 + (spaceSubscription?.blockQuota || 0) * 1000;
+  const spaceBlockQuota =
+    defaultFreeBlockQuota * 1000 + (spaceSubscription?.blockQuota || 0) * 1000 + additionalBlockQuota;
```
Comment (valentinludu) [inline components/settings/subscription/hooks/useSpaceSubscription.ts:38]: if defaultBlockQuota and spaceSubscription.blockQuota are in the form of 30(K), then I suggest to make additionalBlockQuota the same and multiply it here by 1000 like the rest of them.

**Review by valentinludu — APPROVED** [review]
Looks good. Added some small stuff.

File: components/settings/subscription/hooks/useSpaceSubscription.ts (hunk 1 above)
Comment (Devorein) [reply components/settings/subscription/hooks/useSpaceSubscription.ts:38]: Good point, will update to reflect `spaceSubscription.blockQuota` using thousands

File: components/settings/subscription/hooks/useAdditionalBlockQuota.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export function useAdditionalBlockQuota() {
+  const { space: currentSpace } = useCurrentSpace();
+  const { user } = useUser(); // a user session is required to get a result
+
+  const {
+    data: additionalBlockQuota = 0,
+    isLoading,
+    mutate: refetchAdditionalBlockCount
+  } = useSWR(currentSpace && user ? `/spaces/${currentSpace.id}/additional-block-quota` : null, () =>
```
[the remark this answers is not in the document]
Comment (Devorein) [reply components/settings/subscription/hooks/useAdditionalBlockQuota.ts:15]: Matt suggested totally getting rid of this endpoint and reusing the block-count endpoint to retrieve the additional quota. So we reused `useBlockCount` hook.

**Pull request merged**